File: c/udemy/4-condicional/multiplos.c
//  (adaptado de URI 1044) Fazer um programa para ler dois números inteiros, e dizer se um número é múltiplo do outro. Os números podem ser digitados em qualquer ordem

#include <stdio.h>

int main() {

    int n1, n2;

    printf("Digite dois números inteiros:\n");
    scanf("%d %d", &n1, &n2);

    if (n1 % n2 == 0 || n2 % n1 == 0){
        printf("Múltiplos");
    }
    else{
        printf("Não são múltiplos");
    }

    return 0;
}

File: python/udemy/4-estrutura-condicional/multiplos.py
# (adaptado de URI 1044) Fazer um programa para ler dois números inteiros, e dizer se um número é múltiplo do outro. Os números podem ser digitados em qualquer ordem.

x = int(input('Primeiro número inteiro: '))
y = int(input('Segundo número inteiro: '))

if (x % y == 0) or (y % x == 0):
    print('são múltiplos')
else:
    print('não são múltiplos')